src/Http/Http.Extensions/src/TagsAttribute.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.AspNetCore.Http.Metadata;
+
+namespace Microsoft.AspNetCore.Http
+{
+    /// <summary>
+    /// Specifies a collection of tags in <see cref="Endpoint.Metadata"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Delegate | AttributeTargets.Class, Inherited = false, AllowMultiple = false)]
+    public sealed class TagsAttribute : Attribute, ITagsMetadata
+    {
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="tags"></param>
```

Comment:
```suggestion
        /// <summary>
        /// Initializes an instance of the <see cref="TagsAttribute"/>.
        /// </summary>
        /// <param name="tags">The tags associated with the endpoint.</param>
```

Feel free to improve on this.